### PR TITLE
Upgrade mathjax 846

### LINKED ISSUE #846: \vec and \hat rendered with offset in mathmode
I find when rendering math (either in-line or display mode with $$) my vector and hat symbols show up as offset. For instance, when I use the following text
`
$$ \int_V{ n(\vec{r}, E, \hat{\Omega}, t) \; d^3r }$$
`
I get the following output in the display:
screen shot 2017-08-31 at 1 51 34 pm

### PR DESCRIPTION
**Pull request by adamwulf**
Fixes how \vec and \hat are rendered by upgrading MathJax to 2.7.3.

Closes #846.
Closes #930.

### CODE REVIEW

**Comment by uranusjr** [comment]
What is the relation between this and #930? I see some identical changes in both pull requests. Also, same as #930, please fix code formatting issues. I really want to merge these changes.

**Comment by adamwulf** [comment]
This forks off of #930 to upgrade to 2.7.3, I wasn't sure the best way to send a pull request since it's based off of the other pull request, and I didn't want to base it off of master as the MathJax.js file needed to be updated as well.

I just now pushed up a commit that I think fixes the rest of the lingering formatting issues.

Thanks!

**Comment by uranusjr** [comment]
Cool, so I can merge this one once it’s done and just discard #930, right?

**Comment by adamwulf** [comment]
Yep, that should work - after this one is merged in, then #930 won't be needed as everything in there is included in here too.

73ee4fa includes all the latest formatting changes, so if you see anything else I need to change just lmk :)

File: MacDown/Code/Document/MPDocument.m [hunk 1]
```diff
@@ -172,7 +172,7 @@ - (int)rendererFlags
 @interface MPDocument ()
     <NSSplitViewDelegate, NSTextViewDelegate,
 #if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101100
-     WebEditingDelegate, WebFrameLoadDelegate, WebPolicyDelegate,
+     WebEditingDelegate, WebFrameLoadDelegate, WebPolicyDelegate,WebResourceLoadDelegate,
```
Comment (uranusjr) [inline MacDown/Code/Document/MPDocument.m]: Add an extra space

File: MacDown/Code/Document/MPDocument.m [hunk 2]
```diff
@@ -815,6 +818,23 @@ - (BOOL)textViewShouldMoveToLeftEndOfLine:(NSTextView *)textView
...
+- (NSURLRequest *)webView:(WebView *)sender resource:(id)identifier willSendRequest:(NSURLRequest *)request redirectResponse:(NSURLResponse *)redirectResponse fromDataSource:(WebDataSource *)dataSource
+{
+    
+    if ([[request.URL lastPathComponent] isEqualToString:@"MathJax.js"])
+    {
+        NSURLComponents *origComps = [NSURLComponents componentsWithURL:[request URL] resolvingAgainstBaseURL:YES];
+        NSURLComponents *updatedComps = [NSURLComponents componentsWithURL:[[NSBundle mainBundle] URLForResource:@"MathJax" withExtension:@"js" subdirectory:@"MathJax"] resolvingAgainstBaseURL:NO];
+        [updatedComps setQueryItems:[origComps queryItems]];
+        
+        request = [NSMutableURLRequest requestWithURL:[updatedComps URL]];
```
Comment (uranusjr) [inline MacDown/Code/Document/MPDocument.m]: Does this needs to be mutable? I feel an `NSURLRequest` would be sufficient.

**Comment by adamwulf** [comment]
done :)

**Pull request merged**